terraform/modules/iam_role/concourse_worker/role.tf

```diff
@@ -43,7 +43,8 @@ module "concourse" {
         "s3:PutObject"
       ],
       "Resource": [
-        "arn:${var.aws_partition}:s3:::cloud-gov-varz/master-bosh-state.json"
+        "arn:${var.aws_partition}:s3:::cloud-gov-varz/master-bosh-state.json",
+        "arn:${var.aws_partition}:s3:::terraform-state/*"
```

Comment:
and probably on the bucket itself as well `arn:${var.aws_partition}:s3:::terraform-state`.

Should we be passing these bucket names in as variables rather than hardcoding them?